web/src/components/pages/contribution/speak/recording-pill.css

```diff
@@ -50,3 +50,7 @@
         width: 55px;
     }
 }
+
+.tippy-popper .tippy-tooltip {
+    max-width: 200px;
+}
```

Comment:
I think it will be better to change this style [here](https://github.com/common-voice/common-voice/blob/412d4bec1a7e5d37977e02496e84784e64609150/web/src/components/index.css#L313)

Just update the 140px to 200px like you have here (and remove this change here)

Reply:
I made changes to the correct file and thanks for the review.